configs/mvp/campus_config/mvp_campus.py

```diff
@@ -1,5 +1,6 @@
 # finetune with the weight pretrained from panoptic dataset
 # using cam3, cam12, cam23
+# yapf: disable
```

Comment:
If `noqa` is used below, we may not need `yapf: disable` anymore?

Reply:
Fixed in [bb71f8e](https://github.com/openxrlab/xrmocap/pull/29/commits/bb71f8eef16e36d321b7ee98c8689cb11bdda78c).